src/component/strategies/CreateStrategy/CreateStrategy.tsx

```diff
@@ -74,6 +74,7 @@ export const CreateStrategy = () => {
             description="The strategy type and the parameters will be selectable when adding an activation strategy to a toggle in the environments.
             The parameter defines the type of activation strategy. E.g. you can create a type 'Teams' and add a parameter 'List'. Then it's easy to add team names to the activation strategy"
             documentationLink="https://docs.getunleash.io/advanced/custom_activation_strategy"
+            documentationLinkLabel="More about custom strategies"
```

Comment:
Should this be "custom _activation_ strategies", I wonder? Would that be clearer? Or does it not matter?

```suggestion
            documentationLinkLabel="More about custom activation strategies"
```